index.html

```diff
@@ -1,18 +1,92 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
     <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <div class="container">
+      <nav>
+        <div>
+          <img src="images/spotify-logo.png" width="260px">
+        </div>
+  
+        <div class="boxes">
+          <a class="box">Premium</a>
+          <a class="box">Discover</a>
+          <a class="box">Help</a>
+          <a class="box">Download</a>
+        </div>
+      </nav>
+  
+      <section class="bigpicture">
```

Comment:
Good to see you using semantic ```section``` HTML tag. I encourage you to use them more for the different sections you create on your pages 👍🏾